### ISSUE TITLE
JSON Inspections failure

### ISSUE THREAD

**Issue by Davoleo** [DemonWav/MinecraftDevIntelliJ#726]
Didn't see any option related to JSON inspections in IntelliJ's Settings 

Please include the following information in all bug reports:
* Minecraft Development for IntelliJ plugin version: 1.4.0
* IntelliJ version: 2020.1.2
* Operating System: Windows 10 1909

Stacktrace:
```
Error while indexing D:\Everything\cache\.gradle\caches\minecraft\deobfedDeps\deobf\constructs-armory\conarm\1.12.2\conarm-1.12.2.jar!\assets\conarm\lang\book\pt_br\intro\welcome.json
To reindex this file IDEA has to be restarted

java.lang.IllegalStateException
	at com.google.gson.JsonArray.getAsString(JsonArray.java:226)
	at com.demonwav.mcdev.translations.index.JsonTranslationProvider.map(providers.kt:59)
	at com.demonwav.mcdev.translations.index.TranslationIndex$Indexer.map(TranslationIndex.kt:111)
	at com.demonwav.mcdev.translations.index.TranslationIndex$Indexer.map(TranslationIndex.kt:108)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapByIndexer(MapReduceIndex.java:317)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapInput(MapReduceIndex.java:309)
	at com.intellij.util.indexing.VfsAwareMapReduceIndex.mapInput(VfsAwareMapReduceIndex.java:150)
	at com.intellij.util.indexing.VfsAwareMapReduceIndex.mapInput(VfsAwareMapReduceIndex.java:40)
	at com.intellij.util.indexing.impl.MapReduceIndex.calculateUpdateData(MapReduceIndex.java:267)
	at com.intellij.util.indexing.impl.MapReduceIndex.update(MapReduceIndex.java:239)
	at com.intellij.util.indexing.FileBasedIndexImpl.updateSingleIndex(FileBasedIndexImpl.java:1253)
	at com.intellij.util.indexing.FileBasedIndexImpl.lambda$doIndexFileContent$20(FileBasedIndexImpl.java:1181)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.freezeFileTypeTemporarilyIn(FileTypeManagerImpl.java:723)
	at com.intellij.util.indexing.FileBasedIndexImpl.doIndexFileContent(FileBasedIndexImpl.java:1161)
	at com.intellij.util.indexing.FileBasedIndexImpl.indexFileContent(FileBasedIndexImpl.java:1147)
	at com.intellij.util.indexing.UnindexedFilesUpdater.lambda$indexFiles$2(UnindexedFilesUpdater.java:130)
	at com.intellij.openapi.project.CacheUpdateRunner.lambda$null$1(CacheUpdateRunner.java:208)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.openapi.project.CacheUpdateRunner.lambda$null$2(CacheUpdateRunner.java:224)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:166)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:627)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:572)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:153)
	at com.intellij.openapi.project.CacheUpdateRunner.lambda$createRunnable$3(CacheUpdateRunner.java:221)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:210)
	at com.intellij.util.ConcurrencyUtil.lambda$underThreadNameRunnable$3(ConcurrencyUtil.java:198)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:20)
	at com.intellij.util.RunnableCallable.call(RunnableCallable.java:11)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.call(ApplicationImpl.java:255)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by RedNesto** [comment]
It seems the I18N index needs to be stricter about what files it indexes

**Comment by Davoleo** [comment]
oh, that's why I'm getting this error with all kinds of different files 🤔

**Comment by RedNesto** [comment]
Well, I guess regardless of the MC version the indexer reads all language JSON files under the assets/modid/lang folder, I guess we should catch the exception and only index JSON files if MCP version is 1.13 or later

**Comment by RedNesto** [comment]
Fixed in d9de3ef020b9b84c753122ced38328f22c99e8ad
